src/app/shared/utils/health-status.ts

```diff
@@ -24,6 +24,8 @@ export enum StatusIcon {
   Error = 'km-icon-error',
   Disabled = 'km-icon-disabled',
   Unknown = 'km-icon-unknown',
+  Warning = 'km-icon-warning-state',
```

Comment:
can you rename to `km-icon-warning` only don't add `state` to make it consistent with other enum values

Reply:
ah! cool i can see with `danger icon and !` okay leave it.